INSTALL.md

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Is the extra newline needed for visibility?

Reply:
yes to avoid being hidden by the enter token box